src/scripts/main.js

```diff
@@ -1,3 +1,104 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const buttons = [...document.querySelectorAll('.button')];
+const table = document.querySelector('.field');
+const [tableBody] = table.tBodies;
+
+const MAX = 10;
+const MIN = 2;
+
+const APPEND_ROW = 'append-row';
+const REMOVE_ROW = 'remove-row';
+const APPEND_COLUMN = 'append-column';
+const REMOVE_COLUMN = 'remove-column';
+
+const appendRow = () => {
+  const newRow = tableBody.lastElementChild.cloneNode(true);
+
+  tableBody.append(newRow);
+};
+
+const removeRow = () => {
+  const rowToDelete = tableBody.lastElementChild;
+
+  rowToDelete.remove();
+};
+
+const appendColumn = () => {
+  [...table.rows].forEach(row => {
+    const newCell = row.lastElementChild.cloneNode(true);
+
+    row.append(newCell);
+  });
+};
+
+const removeColumn = () => {
+  [...table.rows].forEach(row => {
+    row.lastElementChild.remove();
+  });
+};
```

Comment:
would be better to add `if` statements and check on max and min, because user can remove `disabled` attribute in inspector and break your logic, send a lot of requests to db etc)
<img width="1240" alt="Screenshot 2023-08-13 at 7 34 19 PM" src="https://github.com/mate-academy/js_growth_table_DOM/assets/60295799/10f4ce26-622d-47ab-a7c3-33b18b569cd3">